src/lib.rs

```diff
@@ -392,6 +393,14 @@ impl<'a> DeployData<'a> {
         };
         Ok(profile_user)
     }
+
+    fn get_sudo(&'a self) -> Result<String, DeployDataDefsError> {
```

Comment:
Does this need to be a `Result` type? I don't think there can be any errors to return here :thinking:

Reply:
No, it doesn't. And if it had to be a `Result`, the `Error` type should have been different anyways :)